examples/ex9.cpp

```diff
@@ -237,7 +251,7 @@ int main(int argc, char *argv[])
 
```

Comment:
```suggestion
   if (pa && mesh.GetNodes() != NULL)
   {
      mesh.SetCurvature(mesh.GetNodalFESpace()->GetOrder(0), true);
   }
```

Something like this should be added to make sure ex9 works with e.g. star-q3

Reply:
done.